cpp/client/ct.cc

```diff
@@ -753,7 +755,7 @@ static void DiagnoseCertChain() {
       continue;
     }
     LOG(INFO) << "SCT number " << i + 1 << ":\n" << sct.DebugString();
-    if (verifier != NULL) {
+    if (verifier.get()) {
```

Comment:
I thought our code style was "if (wibble)", should it be "if (wibble != nullptr)"?